config/api.yml

```diff
@@ -814,6 +814,8 @@
       :post:
       - :name: delete
         :identifier: cloud_volume_delete
+      - :name: safe_delete
+        :identifier: cloud_volume_safe_delete
```

Comment:
@gregoryb-at-ibm looks like this is failing [here](https://travis-ci.com/github/ManageIQ/manageiq-cross_repo-tests/jobs/484223860#L2479-L2482) because this isn't a product feature

Reply:
@agrare , yeah, I've added the feature.